ofrak_core/test_ofrak/unit/test_ofrak_server.py

```diff
@@ -26,6 +29,14 @@ def hello_world_elf() -> bytes:
     return hello_elf()
 
 
+@pytest.fixture()
+async def large_test_file(ofrak_context: OFRAKContext) -> Resource:
+    with tempfile.NamedTemporaryFile() as temp:
+        for i in range(0, 256, 1):
+            temp.write(int.to_bytes(i, 1, "big") * 4096)
+        yield await ofrak_context.create_root_resource_from_file(temp.name)
```

Comment:
Two minor things:

- I think the suggestion below is equivalent, but a little more concise
- Is this file big enough? By my calculation, it's `4096 * 4 * 256 ≈ 4MB`, but not really on the order of the gigabytes we might be seeing with this changeset

```suggestion
async def large_test_file(ofrak_context: OFRAKContext) -> Resource:
    with tempfile.NamedTemporaryFile() as temp:
        for i in range(256):
            temp.write(int.to_bytes(i, 1, "big") * 4096)
        yield await ofrak_context.create_root_resource_from_file(temp.name)
```